src/main/java/com/maxmind/geoip2/WebServiceClient.java

```diff
@@ -254,19 +261,15 @@ public InsightsResponse insights(InetAddress ipAddress) throws IOException,
 
         String body = WebServiceClient.getSuccessBody(response, uri);
 
-        InjectableValues inject = new InjectableValues.Std().addValue(
-                "locales", this.locales);
-        ObjectMapper mapper = new ObjectMapper();
-        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES,
-                false);
-        mapper.configure(MapperFeature.CAN_OVERRIDE_ACCESS_MODIFIERS, false);
+        String ip = ipAddress == null ? null : ipAddress.getHostAddress();
```

Comment:
The web service will always return an IP address on a successful request. We should just use that value.